components/picker/index.css

```diff
@@ -10,117 +10,185 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+  --spectrum-picker-spacing-icon-to-disclosure-icon: var(--spectrum-picker-visual-to-disclosure-icon-extra-large);
+  --spectrum-picker-spacing-label-to-picker-quiet: var(--spectrum-field-label-to-component-quiet-extra-large);
+  --spectrum-picker-spacing-picker-to-popover: var(--spectrum-component-to-menu-extra-large);
 }
 
-.spectrum-Picker {
-  /* todo: remove when fixed in DNA */
-  --spectrum-picker-texticon-min-width: var(--spectrum-global-dimension-size-400);
+/* Windows high contrast mode */
+@media (forced-colors: active) {
+  .spectrum-Picker {
+    --highcontrast-picker-focus-indicator-color: Highlight;
 
-  /* Todo: fix in DNA, should have been zero */
-  --spectrum-picker-texticon-disabled-border-size: 0;
+    --highcontrast-picker-border-color-default: ButtonText;
+    --highcontrast-picker-border-color-active: ButtonText;
+    --highcontrast-picker-border-color-error-active: ButtonText;
+    --highcontrast-picker-border-color-key-focus: Highlight;
 
-  --spectrum-picker-texticon-popover-max-width: var(--spectrum-global-dimension-size-3000);
-  --spectrum-picker-texticon-width: var(--spectrum-global-dimension-size-2400);
+    --highcontrast-picker-font-color-key-focus: ButtonText;
+    --highcontrast-picker-font-color-default: ButtonText;
+    --highcontrast-picker-font-color-disabled: GrayText;
 
-  /* todo: use real DNA var */
-  --spectrum-picker-texticon-border-size-increase-focus: 1px;
+    --highcontrast-picker-background-color-default: Background;
+    --highcontrast-picker-background-color-disabled: Background;
 
-  /* Adjustments for inset/outset padding in DNA */
-  --spectrum-picker-padding-left-adjusted: calc(var(--spectrum-picker-texticon-padding-left) - var(--spectrum-picker-texticon-border-size));
-  --spectrum-picker-padding-right-adjusted: calc(var(--spectrum-picker-texticon-padding-right) - var(--spectrum-picker-texticon-border-size));
-  --spectrum-picker-textonly-padding-left-adjusted: calc(var(--spectrum-picker-textonly-padding-left) - var(--spectrum-picker-textonly-border-size));
-  --spectrum-picker-textonly-padding-right-adjusted: calc(var(--spectrum-picker-textonly-padding-right) - var(--spectrum-picker-textonly-border-size));
-  --spectrum-picker-focus-ring-border-radius-adjusted: calc(var(--spectrum-picker-textonly-border-radius) + var(--spectrum-picker-focus-ring-gap));
+    --highcontrast-picker-icon-color-default: ButtonText;
+    --highcontrast-picker-icon-color-hover: ButtonText;
+    --highcontrast-picker-icon-color-error-default: ButtonText;
+    --highcontrast-picker-icon-color-key-focus: Highlight;
+
+    &:disabled,
+    .is-disabled {
+      border-color: GrayText;
+      border-width: var(--spectrum-picker-border-size);
+    }
+  }
+  .spectrum-Picker--quiet {
+    &:focus-ring,
+    &:focus-visible,
+    &.is-focused {
+      forced-color-adjust: none;
+      outline: 0;
+      box-shadow: 0 1px 0 0 var(--spectrum-picker-focus-indicator-color);
```

Comment:
Is this double line the desired effect here?

<img width="144" alt="Screen Shot 2022-10-31 at 11 00 27 AM" src="https://user-images.githubusercontent.com/25614178/199039755-3ab140c2-63be-4d82-984c-ce5cb44812b8.png">
<img width="141" alt="Screen Shot 2022-10-31 at 11 00 54 AM" src="https://user-images.githubusercontent.com/25614178/199039758-4aa9059f-beba-44eb-b718-8f6a2d4eb548.png">

Reply:
This should be fixed now.